contracts/NFTOpt.sol

```diff
@@ -152,10 +152,11 @@ contract NFTOpt {
     external
     payable
     {
-        // TODO: update this with the correct implementation (wrote this here only for testing)
+        require (options[_optionId].buyer == msg.sender , "Only buyer can withdraw option request");

```

Comment:
```
delete options[_optionsID];
```

instead of line 156

Reply:
Can you clarify?  Unclear what you mean

Reply:
There's no context for referencing the buyer, would need to add something like:
Option memory option = options[_optionId];
then use
option.buyer
is that what you mean?

Reply:
i mean exactly what was said

I'll restate:

replace line 156 with 

```delete options[_optionsID];```

Reply:
oh, I see what you're getting at, thanks.  I had it in my head that this needed to be state closed.